src/functions/functions.ts

```diff
@@ -1,40 +1,67 @@
-﻿function add(first: number, second: number): number {
+﻿/**
+ * Adds two numbers.
+ * @param first First number
+ * @param second Second number
+ * @returns The sum of the two numbers.
+ */
+function add(first: number, second: number): number {
   return first + second;
 }
 
-function clock(callback) {
+/**
+ * Displays the current time once a second.
+ * @param handler Custom function handler  
+ */
+function clock(handler: CustomFunctions.StreamingHandler<string>) {
   const timer = setInterval(() => {
     const time = currentTime();
-    callback.setResult(time);
+    handler.setResult(time);
   }, 1000);
 
-  callback.onCanceled = () => {
+  handler.onCanceled = () => {
     clearInterval(timer);
   };
 }
 
+/**
+ * Returns the current time.
+ * @returns String with the current time formatted according to the current locale conventions.
+ */
 function currentTime() {
```

Comment:
should we have a return type here: _function currentTime() : string { … }_

Reply:
Yes, I was just making updates for the JSDoc comments and I will add this.

Reply:
@TCourtneyOwen I pushed an update.